lib/manifold.ex

```diff
@@ -26,12 +27,33 @@ defmodule Manifold do
   @spec send([pid | nil] | pid | nil, term) :: :ok
   def send([pid], message), do: __MODULE__.send(pid, message)
   def send(pids, message) when is_list(pids) do
+    partitioner_name = current_partitioner()
     grouped_by = Utils.group_by(pids, fn
       nil -> nil
       pid -> node(pid)
     end)
-    for {node, pids} <- grouped_by, node != nil, do: Partitioner.send({Partitioner, node}, pids, message)
+    for {node, pids} <- grouped_by, node != nil, do: Partitioner.send({partitioner_name, node}, pids, message)
+    :ok
   end
-  def send(pid, message) when is_pid(pid), do: Partitioner.send({Partitioner, node(pid)}, [pid], message)
+  def send(pid, message) when is_pid(pid), do: Partitioner.send({current_partitioner(), node(pid)}, [pid], message)
   def send(nil, _message), do: :ok
+
+  def current_partitioner() do
+    partitioner_for(self())
```

Comment:
Do we think this can ever have a situation where during a handoff stuff starts being sent to another partitioner out of order?

Reply:
no, there is no linearization requirement for handoffs across processes 👍

Reply:
Are you sure?

If a manifold is backed up on a server.. then there is a handoff on another service which changes the current pid.. you might send things to a new partitioner on the backed up server, causing messages to arrive out of order?